libs/datasets/timeseries.py

```diff
@@ -105,12 +107,14 @@ def get_subset(self, after=None, columns=tuple()):
         rows_key = dataset_utils.make_rows_key(self.data, after=after,)
         columns_key = list(columns) if columns else slice(None, None, None)
         return OneRegionTimeseriesDataset(
-            self.data.loc[rows_key, columns_key].reset_index(drop=True), latest=self.latest,
+            self.data.loc[rows_key, columns_key].reset_index(drop=True),
+            latest=self.latest,
+            provenance=self.provenance,
```

Comment:
`latest` and `provenance` are going to have columns outside of the `columns` subset specified, right?  But maybe we don't care?  Perhaps at least add a comment.

Reply:
Maybe, but probably not soon.